test/extended/networking.sh

```diff
@@ -189,7 +214,7 @@ ${OPENSHIFT_INSTANCE_PREFIX:-openshift}"
     fi
     ;;
   *)
-    if [ "${CONFIG_ROOT}" != "" ]; then
+    if [ -n "${CONFIG_ROOT}" ]; then
```

Comment:
re: [[ vs [ - Done.

Regarding CONFIG_ROOT, it's an alias to OPENSHIFT_CONFIG_ROOT and it's initialized on line 202: 

https://github.com/openshift/origin/pull/8506/files#diff-71ec4266e3a35d109f56f03dc3cd3ed9R202